dockerfiles/Dockerfile.cuda

```diff
@@ -17,10 +17,9 @@ RUN apt-get update &&\
 WORKDIR /code
 ENV PATH /usr/local/nvidia/bin:/usr/local/cuda/bin:/code/cmake-3.14.3-Linux-x86_64/bin:/opt/miniconda/bin:${PATH}
 
-# Prepare onnxruntime repository & build onnxruntime with TensorRT
-ADD scripts /tmp/scripts
-RUN /bin/sh /tmp/scripts/install_common_deps.sh && \
-    git clone --single-branch --branch ${ONNXRUNTIME_SERVER_BRANCH} --recursive ${ONNXRUNTIME_REPO} onnxruntime &&\
+# Prepare onnxruntime repository & build onnxruntime with CUDA
+RUN git clone --single-branch --branch ${ONNXRUNTIME_SERVER_BRANCH} --recursive ${ONNXRUNTIME_REPO} onnxruntime &&\
+    /bin/sh onnxruntime/dockerfiles/install_common_deps.sh &&\
```

Comment:
is this ...dockerfiles/**scripts**/install_common_deps.sh?

Reply:
Updated! Looks like it got mixed up in the merging process.